src/main/java/org/folio/converter/Field008SplitterFactory.java

```diff
@@ -0,0 +1,218 @@
+package org.folio.converter;
+
+import static org.folio.converter.StringConstants.ACCM;
```

Comment:
Reduce static imports to import static org.folio.converter.StringConstants.*

Reply:
done